src/home/Benefit.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 import './Benefit.css'
-
+import  Benefit1 from "../assets/images/benefit1.png"
+import  Benefit2 from "../assets/images/benefit2.png"
 
 const Benefit = () => {
     return (
-        <div className="aboutContainer">
+        <div className="container">
```

Comment:
@purnima143 , with this name change, you also need to change the reference being used in the Benefit.css file (instead of `aboutContainer` now it should be `container`). 
![Screen Shot 2020-12-16 at 12 48 20 pm](https://user-images.githubusercontent.com/29667122/102294257-0159a680-3f9d-11eb-83d8-bc294e4e9ab1.png)

Atm, the container style seems to be off since the container is not using the .aboutContainer values.

<img width="1617" alt="Screen Shot 2020-12-16 at 12 50 14 pm" src="https://user-images.githubusercontent.com/29667122/102294408-5a293f00-3f9d-11eb-8a93-4feae5e6588f.png">

Reply:
@mtreacy002  Done, Please merge it